test failure in pennylane  autoray ` AttributeError: 'int' object has no attribute 'ndim'`

See for example https://github.com/CQCL/pytket-pennylane/actions/runs/4350531114/jobs/7604916189 

python3.9/site-packages/autoray/autoray.py:886: in ndim
    return x.ndim
E   AttributeError: 'int' object has no attribute 'ndim'

--- a/docs/changelog.rst
+++ b/docs/changelog.rst
@@ -1,6 +1,12 @@
 Changelog
 ~~~~~~~~~
 
+Unreleased
+----------
+
+* Updated pennylane version requirement to 0.30.
+* Updated pytket version requirement to 1.14.
+
 0.7.0 (January 2023)
 --------------------
 

--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@ setup(
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.11",
+        "pytket ~= 1.14",
         "pennylane ~= 0.30.0",
         "pytket-qiskit ~= 0.32.0",
     ],

--- a/setup.py
+++ b/setup.py
@@ -33,7 +33,7 @@ setup(
     include_package_data=True,
     install_requires=[
         "pytket ~= 1.11",
-        "pennylane ~= 0.28.0",
+        "pennylane ~= 0.30.0",
         "pytket-qiskit ~= 0.32.0",
     ],
     classifiers=[

--- a/tests/test_backends.py
+++ b/tests/test_backends.py
@@ -24,7 +24,7 @@ def my_quantum_function(x, y):  # type: ignore
     qml.RX(y, wires=1)
     qml.CNOT(wires=[0, 1])
     qml.RY(y, wires=1)
-    qml.T(wires=0).inv()
+    qml.adjoint(qml.T(wires=0))
     qml.S(wires=0)
     qml.U1(x, wires=0)
     qml.U2(x, y, wires=0)

